tools/boxstarter/node_boxstarter_vscode

```diff
@@ -0,0 +1,25 @@
+# Boxstarter (http://boxstarter.org/) script for Node.js prerequisites,
+#  including Visual Studio Code
+#
+# To install either open this link in IE or Edge:
+#  http://boxstarter.org/package/nr/url?https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter_vscode
+#
+# Or run those commands in a Powershell terminal:
+#  iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1'))
+#  get-boxstarter -Force
+#  Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/tools/boxstarter/node_boxstarter_vscode -DisableReboots
+#
+# For more detail see
+# https://github.com/nodejs/node/blob/master/tools/boxstarter/README.md
+#
+
+# Git and Unix tools will be added to the PATH
+choco install git -params /GitAndUnixToolsOnPath -y
+choco install python2 -y
+
+# Installs VS 2017 Build Tools
+choco install visualstudio2017buildtools -y
+choco install visualstudio2017-workload-vctools -y
+
+# Installs Visual Studio Code
+choco install visualstudiocode -y
```

Comment:
+1 to not installing an editor, especially if it means duplicating the whole script.

Reply:
Instead of duplicating the script, we could just check if an environment variable is set indicating whether an editor should be installed? (not advocating one way or the other but just pointing out that we likely don't need to duplicate the whole script)